target-riscv/cpu.c

```diff
@@ -96,16 +92,7 @@ static void riscv_cpu_disas_set_info(CPUState *s, disassemble_info *info) {
 static void riscv_cpu_realizefn(DeviceState *dev, Error **errp)
 {
     CPUState *cs = CPU(dev);
-    RISCVCPU *cpu = RISCV_CPU(dev);
     RISCVCPUClass *mcc = RISCV_CPU_GET_CLASS(dev);
-    CPURISCVState *env = &cpu->env;
-
-    /* Enable GC ISA */
-    set_feature(env, RISCV_FEATURE_RVM);
```

Comment:
@sorear: sorry, I didn't mean my comment to read as an objection. I was responding to Bastian to point out that this sort of thing is not uncommon amongst other architectures. I was also trying to clarify that supporting `misa` writes isn't essential, but as you point out there are advantages to supporting as much as possible.